### PR TITLE
Fix audio player

### PR DESCRIPTION
**Pull request by jkelvie**

### CODE REVIEW

File: src/SkillInteractor.ts [hunk 1]
```diff
@@ -79,17 +79,20 @@ export abstract class SkillInteractor {
      * Passes in an intent with slots as a simple JSON map: {slot1: "value", slot2: "value2", etc.}
      * @param intentName
      * @param slots
-     * @param callback
      */
-    public intended(intentName: string, slots?: any, requestFilter?: RequestFilter): Promise<any> {
+    public intended(intentName: string, slots?: any): Promise<any> {
         try {
-            return this.callSkillWithIntent(intentName, slots, requestFilter);
+            return this.callSkillWithIntent(intentName, slots);
```
Comment (jperata) [inline src/SkillInteractor.ts]: in order for the try to catch the error here, you have to do 
`return await this.callSkillWithIntent(intentName, slots);`

File: test/AudioPlayerTest.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        try {
+            // We capture the requests being sent to virtual alexa
+            // Because the AudioPlayer does some stuff internally automatically, want to ensure it is working properly
+            const requests: any[] = [];
+            virtualAlexa.filter((json) => {
+                requests.push(json.request);
+            });
+
+            let reply = await virtualAlexa.launch();
+            reply = await virtualAlexa.utter("play");
```
Comment (jperata) [inline test/AudioPlayerTest.ts]: await virtualAlexa.launch();
const reply = await virtualAlexa.utter("play");

File: test/resources/SimpleAudioPlayer.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
+// This is only initialized when the Lambda is, so it is preserved across calls
+// It is NOT a real database, but can be used for testing, as JavaScript Lambdas tend to live for a few hours
+// Stay tuned for a more sophisticated example that uses DynamoDB
+var lastPlayedByUser = {};
```
Comment (jperata) [inline test/resources/SimpleAudioPlayer.js]: Is this class using var for any particular reason? , if not using const and let's for consistency make sense

Comment (jkelvie) [reply test/resources/SimpleAudioPlayer.js]: This is an exact copy of our simple audio player -  unfortunately, I use the old var style there, and as I would like to keep these in sync, changing this does not make sense.

File: src/SkillInteractor.ts (hunk 1 above)
Comment (jkelvie) [reply src/SkillInteractor.ts]: I added that try/catch for debugging - removed it

**Pull request merged**